app/controllers/reports/instrument_unavailable_export_raw_reports_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Reports
+
+  class InstrumentUnavailableExportRawReportsController < ReportsController
+
+    include CsvExportController
```

Comment:
Would it make more sense to have `CsvExportController` be a class and have it inherit from `ReportsController`?